tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/types/string/StringValueBasicsTest.java

```diff
@@ -69,6 +70,13 @@ public void testError() {
         Assert.assertEquals(((BInteger) returns[0]).intValue(), 5);
     }
 
+    @Test
+    public void testArrayStore() {
+        BValue[] returns = BRunUtil.invoke(result, "testArrayStore");
+        Assert.assertEquals(returns[0].getClass(), BValueArray.class);
+        Assert.assertEquals(((BValueArray) returns[0]).getStringArray()[0], "h😀llo");
```

Comment:
We assert the length because that is the main difference from the old String. If we assert by value both old and new cases pass. Shall we return the length and assert?